.travis.yml

```diff
@@ -212,6 +212,3 @@ jobs:
         local_dir: playground/dist
         on:
           branch: master
-
```

Comment:
I think this landed with the related work done by @hypest.

Reply:
Yeah, will have to resolve conflicts later